bedrock/firefox/templates/firefox/sync.html

```diff
@@ -0,0 +1,70 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "macros-protocol.html" import hero, picto_card, feature_card with context %}
+
+{% block page_title %}{{ ftl('firefox-sync') }}{% endblock %}
+
+{% block page_desc %}{{ ftl('access-and-sync') }}{% endblock %}
+
+{% block page_css %}
+ {{ css_bundle('sync') }}
+{% endblock %}
+
+{% block content %}
+<div class="mzp-l-content">
+  {% call hero(
+    title=ftl('sync-your-firefox'),
+    class='mzp-has-image mzp-t-firefox mzp-t-product-firefox',
+    image_url='img/firefox/accounts/trailhead/value-knowledge.jpg',
+    include_highres_image=True,
+    include_cta=True,
+    heading_level=1
+  ) %}
+  <p>{{ ftl('access-and-sync') }}</p>
+  {{ download_firefox_thanks() }}
```

Comment:
It feels a bit weird that this page is all about Sync, yet the primary CTA is still "Download Firefox" when I view the page in Firefox? At first i didn't notice the Sync instructions further down the page. Would it be worth replacing this with a sign-in button for Firefox users?

Reply:
Agreed! Done.